keras_nlp/models/bert.py

```diff
@@ -25,6 +25,27 @@ def _bert_kernel_initializer(stddev=0.02):
     return keras.initializers.TruncatedNormal(stddev=stddev)
 
 
+# Pretrained models
+BASE_PATH = "https://storage.googleapis.com/keras-nlp/models/"
+
+checkpoints = {
+    "bert_base": {
+        "bert_base_uncased": {
```

Comment:
Should we add `en` to the name here? And potentially remove the "bert_base" prefix, given that that become redundant when typing this out in the symbol?

`BertBase(weights="uncased_en")` and `BertLarge(weight="uncase_en")` feel like a cleaner usage for me. And you could find replace on just the API symbol name to switch sizes, which feels like a plus.